CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # [Unreleased](https://github.com/pybamm-team/PyBaMM/)
 
+# [v23.5](https://github.com/pybamm-team/PyBaMM/tree/v23.5) - 2023-06-18
```

Comment:
Will the date here be updated?

Reply:
I was thinking of keeping it as it is the date of the feature freeze